lib/form_data.js

```diff
@@ -162,7 +162,7 @@ FormData.prototype._multiPartHeader = function(field, value, options) {
   // custom header specified (as string)?
   // it becomes responsible for boundary
   // (e.g. to handle extra CRLFs on .NET servers)
-  if (options.header) {
+  if (typeof options.header === 'string') {
```

Comment:
Do you mind to follow style we have in the code base,
I mean non-strict equals where it's not necessary. :)